frontend/src/components/Appbar.tsx

```diff
@@ -38,12 +38,19 @@ function ProfileBox() {
     localStorage.removeItem("token");
     navigate("/");
   };
+
+  const GoToBookmarks = () => {
```

Comment:
function names should be camelcase instead of pascalcase

Reply:
will do! i also changed the name of the Logout function for consistency